src/oemof/solph/components/_generic_storage.py

```diff
@@ -1312,6 +1312,22 @@ def _old_storage_capacity_rule(block):
 
             self.old_rule_build = BuildAction(rule=_old_storage_capacity_rule)
 
+            def _initially_empty_rule(block):
+                """Ensure storage to be empty initially"""
+                for n in self.INVESTSTORAGES:
+                    for t in m.TIMESTEPS:
+                        if t == 0:
+                            expr = self.storage_content[n, 0] == 0
+                            self.initially_empty.add((n, t), expr)
```

Comment:
Is the timestep loop necessary? It should be enough to set this constraint just for `t=0`, since there will always be a time step 0, right?
```
                    expr = self.storage_content[n, 0] == 0
                    self.initially_empty.add((n, 0), expr)
```
Tried it and tests still work

Reply:
Agree. That's better.